Carbon.IIS/Functions/Install-CIisWebsite.ps1

```diff
@@ -98,7 +98,9 @@ function Install-CIisWebsite
         [bool] $ServerAutoStart,
 
         # Return a `Microsoft.Web.Administration.Site` object for the website.
-        [switch] $PassThru
+        [switch] $PassThru,
+
+        [TimeSpan] $Timeout = [TimeSpan]::New(0, 0, 30)

```

Comment:
Would it be simpler to use `New-TimeSpan -Seconds 30`?

Reply:
Either works. If I never need to mock out the default timeout, I'll use `New-TimeSpan`. (I actually didn't know that cmdlet existed.)